rb/lib/selenium/webdriver/chrome/bridge.rb

```diff
@@ -116,7 +114,7 @@ def create_capabilities(opts)
 
           # legacy options - for chromedriver < 17.0.963.0
           caps["chrome.switches"] = chrome_options['args'] if chrome_options.member?('args')
-          %w[binary detach extensions nativeEvents noWebsiteTestingDefaults prefs profile verbose].each do |key|
+          %w(binary detach extensions nativeEvents noWebsiteTestingDefaults prefs profile verbose).each do |key|
```

Comment:
Maybe let's keep the `%w[]` style? It's more explicit about literal being array.